### PR TITLE
Added Test for Newsfeed in Backend

### PR DESCRIPTION
**Pull request by Kaushik1216**
Pull Request for Issue # .

### CODE REVIEW

File: tests/System/integration/administrator/components/com_newsfeeds/Newsfeeds.cy.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+describe('Test in backend that the contacts list', () => {
```
Comment (laoneo) [inline tests/System/integration/administrator/components/com_newsfeeds/Newsfeeds.cy.js]: contact?

**Comment by laoneo** [comment]
There is still an error, click on the artefacts link.

**Comment by Kaushik1216** [comment]
working fine in my local system

**Comment by Kaushik1216** [comment]
Test is correct because when I try to add a new newsfeed with Database postgrep from adminstrator panel then also error is showing

Screenshot (138)


Please check why this is happening

**Comment by laoneo** [comment]
Looks like you actually found a bug. @alikon can you help out here what should be done to make it work in postgres?

**Comment by alikon** [comment]
i'll check this evening

**Comment by laoneo** [comment]
@Kaushik1216 can you apply the change to your installation to see if the tests are running trough?

**Comment by Kaushik1216** [comment]
Yes It's working 

Screenshot (139)

**Comment by laoneo** [comment]
Thanks!

**Pull request merged**